update launch , and writing somme agnosticfile template

sifaoufatai opened the pull request.

pep8speaks commented:
Hello @sifaoufatai! Thanks for opening this PR. 

* In the file [`bep32v01/agnostic_file_template.py`](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py):

> [Line 2:3](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L2): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 3:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L3): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 8:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L8): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 13:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L13): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 13:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L13): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (187 > 100 characters)
> [Line 18:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L18): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 18:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L18): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (295 > 100 characters)
> [Line 23:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L23): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 23:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L23): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (150 > 100 characters)
> [Line 28:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L28): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 28:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L28): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (228 > 100 characters)
> [Line 33:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L33): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 38:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L38): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 38:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L38): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (128 > 100 characters)
> [Line 43:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L43): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 43:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L43): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (240 > 100 characters)
> [Line 48:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L48): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 53:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L53): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 53:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L53): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (111 > 100 characters)
> [Line 58:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L58): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 58:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L58): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (134 > 100 characters)
> [Line 63:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L63): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 63:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L63): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (189 > 100 characters)
> [Line 68:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L68): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 73:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L73): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 73:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L73): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (180 > 100 characters)
> [Line 81:3](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L81): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 82:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L82): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 82:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L82): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (259 > 100 characters)
> [Line 87:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L87): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 87:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L87): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (302 > 100 characters)
> [Line 92:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L92): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 92:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L92): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (209 > 100 characters)
> [Line 97:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L97): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 97:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L97): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (205 > 100 characters)
> [Line 102:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L102): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 102:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L102): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (193 > 100 characters)
> [Line 107:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L107): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 107:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L107): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (181 > 100 characters)
> [Line 112:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L112): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 112:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L112): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (208 > 100 characters)
> [Line 117:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L117): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 123:3](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L123): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 124:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L124): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 128:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L128): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 134:9](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L134): [E126](https://duckduckgo.com/?q=pep8%20E126) continuation line over-indented for hanging indent
> [Line 136:7](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L136): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 140:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L140): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 144:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L144): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 149:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L149): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 156:3](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L156): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 157:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L157): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 159:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L159): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (179 > 100 characters)
> [Line 162:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L162): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 164:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L164): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (186 > 100 characters)
> [Line 167:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L167): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 169:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L169): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (244 > 100 characters)
> [Line 172:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L172): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 174:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L174): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (276 > 100 characters)
> [Line 177:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L177): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 179:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L179): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (160 > 100 characters)
> [Line 182:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/agnostic_file_template.py#L182): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent



* In the file [`bep32v01/template_agnotic_file/agonotic_files template.py`](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py):

> [Line 2:3](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L2): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 3:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L3): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 8:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L8): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 13:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L13): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 13:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L13): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (187 > 100 characters)
> [Line 18:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L18): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 18:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L18): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (295 > 100 characters)
> [Line 23:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L23): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 23:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L23): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (150 > 100 characters)
> [Line 28:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L28): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 28:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L28): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (228 > 100 characters)
> [Line 33:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L33): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 38:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L38): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 38:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L38): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (128 > 100 characters)
> [Line 43:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L43): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 43:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L43): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (240 > 100 characters)
> [Line 48:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L48): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 53:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L53): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 53:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L53): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (111 > 100 characters)
> [Line 58:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L58): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 58:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L58): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (134 > 100 characters)
> [Line 63:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L63): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 63:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L63): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (189 > 100 characters)
> [Line 68:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L68): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 73:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L73): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 73:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L73): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (180 > 100 characters)
> [Line 81:3](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L81): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 82:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L82): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 82:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L82): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (259 > 100 characters)
> [Line 87:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L87): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 87:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L87): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (302 > 100 characters)
> [Line 92:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L92): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 92:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L92): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (209 > 100 characters)
> [Line 97:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L97): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 97:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L97): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (205 > 100 characters)
> [Line 102:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L102): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 102:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L102): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (193 > 100 characters)
> [Line 107:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L107): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 107:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L107): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (181 > 100 characters)
> [Line 112:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L112): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent
> [Line 112:101](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L112): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (208 > 100 characters)
> [Line 117:5](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/template_agnotic_file/agonotic_files template.py#L117): [E121](https://duckduckgo.com/?q=pep8%20E121) continuation line under-indented for hanging indent

* In the file [`bep32v01/tests/TestAgnoticfile.py`](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/tests/TestAgnoticfile.py):

> [Line 12:62](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/tests/TestAgnoticfile.py#L12): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 25:9](https://github.com/INT-NIT/BEP032tools/blob/2306940eb1c72fee6f40903236a7169f23be84ad/bep32v01/tests/TestAgnoticfile.py#L25): [E303](https://duckduckgo.com/?q=pep8%20E303) too many blank lines (2)



Do see the [Hitchhiker's guide to code style](https://goo.gl/hqbW4r)

killianrochet commented on bep32v01/template_agnotic_file/agonotic_files template.py:
Ce fichier ne devrait pas être en .py mais en .json

SylvainTakerkart reviewed (changes requested):
Voilà un premier ensemble de commentaires sur la forme... Je vais regarder l'algorithmie plus tard... T'as déjà de quoi faire ;)

sifaoufatai commented:
This version doesn't exist anymore; I deleted it and implemented another
one.






Le ven. 31 mai 2024 à 15:02, Sylvain Takerkart ***@***.***> a
écrit :

> ***@***.**** requested changes on this pull request.
>
> Voilà un premier ensemble de commentaires sur la forme... Je vais regarder
> l'algorithmie plus tard... T'as déjà de quoi faire ;)
> ------------------------------
>
> On bep32v01/agnostic_file_template.py
> <https://github.com/INT-NIT/BEP032tools/pull/150#discussion_r1622319171>:
>
> this file seems to be a double of another file... only keep one! (cf my
> other comment on this other file)
> ------------------------------
>
> On bep32v01/tests/TestAgnoticfile.py
> <https://github.com/INT-NIT/BEP032tools/pull/150#discussion_r1622320076>:
>
> again, check the spelling of your filename: agnoStic
> ------------------------------
>
> In bep32v01/tests/TestAgnoticfile.py
> <https://github.com/INT-NIT/BEP032tools/pull/150#discussion_r1622322940>:
>
> > @@ -0,0 +1,117 @@
> +import unittest
> +import warnings
> +import os
> +import json
> +import csv
> +import yaml
> +from unittest.mock import patch, mock_open, MagicMock
> +from bep32v01.agnostic_file_template import data_description_json_template, \
>
> you try to import from agnostic_file_template.py... is this why you want
> it to be a .py file???
>
> that file should be a json file; and you should simply read it instead of
> importing from it...
> ------------------------------
>
> On bep32v01/writing_agnoticfile.py
> <https://github.com/INT-NIT/BEP032tools/pull/150#discussion_r1622323826>:
>
> the spelling of the filename should be agnoStic! please check everywhere!
> ------------------------------
>
> In bep32v01/writing_agnoticfile.py
> <https://github.com/INT-NIT/BEP032tools/pull/150#discussion_r1622325503>:
>
> > +import json
> +import os
> +import warnings
> +
> +import yaml
> +
> +from bep32v01.agnostic_file_template import data_description_json_template, Citation_template, \
> +    Sample_template
> +from bep32v01.agnostic_file_template import participant_json_template
> +
> +# Assign the data description template
> +data_description_template = data_description_json_template
> +
> +data_participants_template = participant_json_template
> +# Path to the file containing the BIDS version
> +bidversion_path = 'ressources/schema/BIDS_VERSION'
>
> the spelling should be bids, not bid; please check everywhere!
> ------------------------------
>
> In bep32v01/writing_agnoticfile.py
> <https://github.com/INT-NIT/BEP032tools/pull/150#discussion_r1622340380>:
>
> > +            writer = csv.writer(f, delimiter='\t')
> +            writer.writerow(primary_keys.keys())
> +            writer.writerow(primary_keys.values())
> +    elif path_to_save.endswith('.cff'):
> +        with open(path_to_save, 'w') as f:
> +            yaml.dump(primary_keys, f, default_flow_style=False)
> +            print(path_to_save)
> +            print(primary_keys)
> +    elif path_to_save.endswith('.txt'):
> +        with open(path_to_save, 'w') as f:
> +            for key, value in primary_keys.items():
> +                f.write(f"{key}: {value}\n")
> +    return primary_keys
> +
> +
> +def complete_agnostic_file(output_dir, **kwargs):
>
> rename this function by fill_agnostic_file
> ------------------------------
>
> In bep32v01/writing_agnoticfile.py
> <https://github.com/INT-NIT/BEP032tools/pull/150#discussion_r1622342267>:
>
> > +    primary_keys = {key: template[key]["Requirement Level"] for key in template.keys()}
> +
> +    return primary_keys
> +
> +
> +def get_agnostic_file_arguments(template):
> +    """
> +    Extracts arguments from the template to be used for creating an agnostic file.
> +
> +    :param template: The template containing the data description.
> +    :return: A dictionary with argument keys and default values.
> +    """
> +    return {key: None for key in template.keys()}
> +
> +
> +def write_agnostic_files(path_to_save, template, **kwargs):
>
> you're writing only one file with this function, correct? if yes, put file
> on singular (file, not files, in the function name)
> ------------------------------
>
> On bep32v01/template_agnotic_file/agonotic_files template.py
> <https://github.com/INT-NIT/BEP032tools/pull/150#discussion_r1622318459>:
>
> and also, this file is called: template_agnotic_file/agonotic_files
> template.py
>
>    1. you should correct the spelling so that you write agnoStic (in the
>    directory name and in the filename)
>    2. it seems to be a double of the other file at the root directory...
>    only keep one! (cf my other comment on this other file)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/INT-NIT/BEP032tools/pull/150#pullrequestreview-2090662226>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A33HPUWSC5G22ZZQQSKNL3LZFBYHRAVCNFSM6AAAAABIMZ7YHSVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDAOJQGY3DEMRSGY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

sifaoufatai commented:
And i already made those modifications yet

Le ven. 31 mai 2024 à 15:23, sifaou fatai ***@***.***> a écrit :

> This version doesn't exist anymore; I deleted it and implemented another
> one.
>
>
>
>
>
>
> Le ven. 31 mai 2024 à 15:02, Sylvain Takerkart ***@***.***>
> a écrit :
>
>> ***@***.**** requested changes on this pull request.
>>
>> Voilà un premier ensemble de commentaires sur la forme... Je vais
>> regarder l'algorithmie plus tard... T'as déjà de quoi faire ;)
>> ------------------------------
>>
>> On bep32v01/agnostic_file_template.py
>> <https://github.com/INT-NIT/BEP032tools/pull/150#discussion_r1622319171>:
>>
>> this file seems to be a double of another file... only keep one! (cf my
>> other comment on this other file)
>> ------------------------------
>>
>> On bep32v01/tests/TestAgnoticfile.py
>> <https://github.com/INT-NIT/BEP032tools/pull/150#discussion_r1622320076>:
>>
>> again, check the spelling of your filename: agnoStic
>> ------------------------------
>>
>> In bep32v01/tests/TestAgnoticfile.py
>> <https://github.com/INT-NIT/BEP032tools/pull/150#discussion_r1622322940>:
>>
>> > @@ -0,0 +1,117 @@
>> +import unittest
>> +import warnings
>> +import os
>> +import json
>> +import csv
>> +import yaml
>> +from unittest.mock import patch, mock_open, MagicMock
>> +from bep32v01.agnostic_file_template import data_description_json_template, \
>>
>> you try to import from agnostic_file_template.py... is this why you want
>> it to be a .py file???
>>
>> that file should be a json file; and you should simply read it instead of
>> importing from it...
>> ------------------------------
>>
>> On bep32v01/writing_agnoticfile.py
>> <https://github.com/INT-NIT/BEP032tools/pull/150#discussion_r1622323826>:
>>
>> the spelling of the filename should be agnoStic! please check everywhere!
>> ------------------------------
>>
>> In bep32v01/writing_agnoticfile.py
>> <https://github.com/INT-NIT/BEP032tools/pull/150#discussion_r1622325503>:
>>
>> > +import json
>> +import os
>> +import warnings
>> +
>> +import yaml
>> +
>> +from bep32v01.agnostic_file_template import data_description_json_template, Citation_template, \
>> +    Sample_template
>> +from bep32v01.agnostic_file_template import participant_json_template
>> +
>> +# Assign the data description template
>> +data_description_template = data_description_json_template
>> +
>> +data_participants_template = participant_json_template
>> +# Path to the file containing the BIDS version
>> +bidversion_path = 'ressources/schema/BIDS_VERSION'
>>
>> the spelling should be bids, not bid; please check everywhere!
>> ------------------------------
>>
>> In bep32v01/writing_agnoticfile.py
>> <https://github.com/INT-NIT/BEP032tools/pull/150#discussion_r1622340380>:
>>
>> > +            writer = csv.writer(f, delimiter='\t')
>> +            writer.writerow(primary_keys.keys())
>> +            writer.writerow(primary_keys.values())
>> +    elif path_to_save.endswith('.cff'):
>> +        with open(path_to_save, 'w') as f:
>> +            yaml.dump(primary_keys, f, default_flow_style=False)
>> +            print(path_to_save)
>> +            print(primary_keys)
>> +    elif path_to_save.endswith('.txt'):
>> +        with open(path_to_save, 'w') as f:
>> +            for key, value in primary_keys.items():
>> +                f.write(f"{key}: {value}\n")
>> +    return primary_keys
>> +
>> +
>> +def complete_agnostic_file(output_dir, **kwargs):
>>
>> rename this function by fill_agnostic_file
>> ------------------------------
>>
>> In bep32v01/writing_agnoticfile.py
>> <https://github.com/INT-NIT/BEP032tools/pull/150#discussion_r1622342267>:
>>
>> > +    primary_keys = {key: template[key]["Requirement Level"] for key in template.keys()}
>> +
>> +    return primary_keys
>> +
>> +
>> +def get_agnostic_file_arguments(template):
>> +    """
>> +    Extracts arguments from the template to be used for creating an agnostic file.
>> +
>> +    :param template: The template containing the data description.
>> +    :return: A dictionary with argument keys and default values.
>> +    """
>> +    return {key: None for key in template.keys()}
>> +
>> +
>> +def write_agnostic_files(path_to_save, template, **kwargs):
>>
>> you're writing only one file with this function, correct? if yes, put
>> file on singular (file, not files, in the function name)
>> ------------------------------
>>
>> On bep32v01/template_agnotic_file/agonotic_files template.py
>> <https://github.com/INT-NIT/BEP032tools/pull/150#discussion_r1622318459>:
>>
>> and also, this file is called: template_agnotic_file/agonotic_files
>> template.py
>>
>>    1. you should correct the spelling so that you write agnoStic (in the
>>    directory name and in the filename)
>>    2. it seems to be a double of the other file at the root directory...
>>    only keep one! (cf my other comment on this other file)
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/INT-NIT/BEP032tools/pull/150#pullrequestreview-2090662226>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/A33HPUWSC5G22ZZQQSKNL3LZFBYHRAVCNFSM6AAAAABIMZ7YHSVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDAOJQGY3DEMRSGY>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>

SylvainTakerkart commented:
> And i already made those modifications yet 

ah, ok, I don't know why, I was looking at the wrong commit... I see all your changes now!

let me test it...
